components/ActionSheet/docs/typical-use.md

```diff
@@ -0,0 +1,41 @@
+### Typical use
+
+Create an instance of `MDCActionSheetController` and add actions to it, once actions are added. You can now 
```

Comment:
Looks like ", once actions are added" may not be needed anymore. Consider deleting it.

Reply:
Done